packages/forms/backend/sirius-components-widget-reference/src/main/java/org/eclipse/sirius/components/widget/reference/datafetchers/ReferenceWidgetFeatureDataFetcher.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.widget.reference.datafetchers;
+
+import org.eclipse.sirius.components.annotations.spring.graphql.QueryDataFetcher;
+import org.eclipse.sirius.components.graphql.api.IDataFetcherWithFieldCoordinates;
+import org.eclipse.sirius.components.widget.reference.Feature;
+import org.eclipse.sirius.components.widget.reference.ReferenceWidget;
+
+import graphql.schema.DataFetchingEnvironment;
+
+/**
+ * Data fetcher for Reference.feature.
```

Comment:
Data fetcher for Reference Widget feature.

Reply:
Done